src/components/project.js

```diff
@@ -0,0 +1,198 @@
+import { Header, HeaderLineBelow } from "../styles/Header"
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+import styled from "styled-components"
+import Body from "../styles/Body.js"
```

Comment:
This can just be `"styles/body"` with the aliases. Basically they make it so `src/` is the root directory for imports.